fitbenchmarking/fitting_benchmarking.py

```diff
@@ -154,45 +154,45 @@ def do_fitting_benchmark_one_problem(prob, minimizers, use_errors=True, count=0,
 
     # Get function definitions for the problem - one for each starting point
     function_defs = get_function_definitions(prob)
-    # search for lowest chi2
-    min_sum_err_sq = 1.e20
+
     # Loop over the different starting points
     for user_func in function_defs:
+        # search for lowest chi2
+        min_chi_sq = 1.e100  # used to store min chi-sq value found. initialised to large value
+        best_fit = None
```

Comment:
what does best_fit store?

Reply:
For a function_def, best_fit stores the y and x data of the best fit after testing against all the minimisers provided by the user.